examples/openai-pgvector-run.yaml

```
@@ -0,0 +1,87 @@
+# Example llama-stack configuration for OpenAI inference + PSQL (pgvector) vector index (RAG)
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)